rafay/resource_eks_cluster.go

```diff
@@ -3894,9 +3917,18 @@ func expandSubnetSpec(p []interface{}) AZSubnetMapping {
 		}
 		if v, ok := in["name"].(string); ok && len(v) > 0 {
 			obj[v] = elem2
+			namesFrequency[v]++
+			if namesFrequency[v] == 2 {
```

Comment:
Just do > 1 here? 

Also, just make it a utility function so that its cleaner? 

example: https://github.com/RafaySystems/rafay-common/blob/master/proto/types/infraapiserver/gkecommon/util.go#L101

Reply:
In case 3 subnets are present with same name, > 1 will include 2 values in `duplicateNames` slice.